docs/source/howto/run_pwbands.md

```diff
@@ -0,0 +1,133 @@
+(howto-workflows-pw-bands)=
+
+# Calculate a bands structure
+
+The `PwBandsWorkChain` is designed to compute electronic band structures for a given structure using Quantum ESPRESSO's `pw.x`. It automates the complete workflow: optional structure relaxation, SCF calculation, and bands calculation along high-symmetry k-points paths.
```

Comment:
In line with our documentation [conventions](file:///Users/mbercx/project/qe/git/aiida-quantumespresso/docs/build/html/developer.html#documentation), try to stick to one line per sentence. That makes it easier to understand diff, make comments/code suggestions etc.